src/components/Menu/ReactSelectCustomComponents/Formatters/FormatOptionLabelWithLanguageLogo.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { ReactSelectOption } from '../../react-select-option-types';
+import { imageMap } from '../../LanguageDropdownSelector/image-map';
+
+export const FormatOptionLabelWithLanguageLogo = (languageOption: ReactSelectOption) => {
+  const Component = imageMap[languageOption.value] ?? (() => <></>);
```

Comment:
Move this to a little method called something like `returnNullIfNoLanguageAvailable` and make it return null